fsutils/mkfatfs/Kconfig

```diff
@@ -11,7 +11,7 @@ config FSUTILS_MKFATFS
 	---help---
 		Enables support for the mkfatfs utility
 
-config MKFATFS_BUFFER_ALIGMENT
+config MKFATFS_BUFFER_ALIGNMENT
```

Comment:
This one is a breaking change and needs to be mentioned in the release notes.

Reply:
I add documentation label to this PR, so we can find it when making the release.